### PR TITLE
Add telluric 0.2.0 recipe

### PR DESCRIPTION
**Pull request by Juanlu001**
cc @drnextgis.

### CODE REVIEW

**Comment by Juanlu001** [comment]
This supersedes #5748, now that rasterio is available on the main channels.

**Comment by Juanlu001** [comment]
This depends on dsnparse (https://github.com/conda-forge/staged-recipes/pull/6606).

File: recipes/telluric/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/telluric/meta.yaml]: Please, use the new default
```
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
```

File: recipes/telluric/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
```
Comment (marcelotrevisani) [inline recipes/telluric/meta.yaml]: Please, remove this new line here

File: recipes/telluric/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+test:
+  imports:
+    - telluric
+
+about:
+  home: http://github.com/satellogic/telluric
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (marcelotrevisani) [inline recipes/telluric/meta.yaml]: The ``LICENSE`` file is not being packaged.

Comment (marcelotrevisani) [reply recipes/telluric/meta.yaml]: I opened a PR for that, please see
https://github.com/satellogic/telluric/pull/147

Comment (drnextgis) [reply recipes/telluric/meta.yaml]: Thank you @marcelotrevisani! Merged.

Comment (marcelotrevisani) [reply recipes/telluric/meta.yaml]: @drnextgis , is it possible to generate a tag with this modification?

Comment (drnextgis) [reply recipes/telluric/meta.yaml]: I have updated PR with new version.

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge``!  :)

**Pull request merged**